env_daos

```diff
@@ -10,6 +10,29 @@ export DAOS_DISABLE_REQ_FWD=1
 LOCATION=$1
 
 daospath=/$LOCATION/install
+prereqpath=/$LOCATION/install/prereq/dev
+
+PATH=${prereqpath}/argobots/bin:$PATH
+PATH=${prereqpath}/isal/bin:$PATH
+PATH=${prereqpath}/isal_crypto/bin:$PATH
+PATH=${prereqpath}/mercury/bin:$PATH
+PATH=${prereqpath}/ofi/bin:$PATH
+PATH=${prereqpath}/openpa/bin:$PATH
+PATH=${prereqpath}/pmdk/bin:$PATH
+PATH=${prereqpath}/protobufc/bin:$PATH
+PATH=${prereqpath}/psm2/bin:$PATH
+PATH=${prereqpath}/spdk/bin:$PATH
+
+LD_LIBRARY_PATH=${prereqpath}/argobots/lib:$LD_LIBRARY_PATH
```

Comment:
are you sure all libs are in X/lib instead of X/lib64?

Reply:
Yes, all prereq in lib except psm2.